src/app/profile/duck/operations.js

```diff
@@ -0,0 +1,95 @@
+import axios from 'axios';
+import jwtDecode from 'jwt-decode';
+import { toast } from 'react-toastify';
+import actions from './actions';
+import upload from '../../util/uploadToCloudinary';
+import constants from './constants';
+
+const { viewUserProfile, setUserProfile, setPictureUploadStatus } = actions;
+
+export const pictureUtils = imageUrl => dispatch => {
+  dispatch(
+    setPictureUploadStatus({
+      status: constants.PICTURE_UPDATING,
+    }),
+  );
+
+  upload('profile', imageUrl)
+    .then(newProfileUrl => {
+      dispatch(
+        setPictureUploadStatus({
+          status: constants.UPDATE_SUCCESS,
+          newProfileUrl,
+        }),
+      );
+    })
+    .catch(() => {
+      dispatch(
+        setPictureUploadStatus({
+          status: constants.UPDATE_ERROR,
+        }),
+      );
+    });
+};
+
+export const viewProfile = () => dispatch => {
+  const token = localStorage.getItem('token');
+  const decoded = jwtDecode(token);
+  const url = `https://learnground-api-staging.herokuapp.com/api/v1/user/${
+    decoded.username
+  }`;
+
+  const defaultOptions = {
+    headers: {
+      Authorization: token,
+    },
+  };
+
+  return axios
+    .get(url, defaultOptions)
+    .then(({ data }) => {
+      dispatch(viewUserProfile(data));
+    })
+    .catch(({ response }) => {
+      toast.error(response.data.error, {
+        hideProgressBar: true,
+      });
+    });
+};
+
+const url2 = 'https://learnground-api-staging.herokuapp.com/api/v1/profile';
+
+export const updateUserProfile = (
+  firstName,
+  lastName,
+  interest,
+  bio,
+) => dispatch => {
+  const token = localStorage.getItem('token');
+  const defaultOptions = {
+    headers: {
+      Authorization: token,
+    },
+  };
+  return axios
+    .patch(
+      url2,
```

Comment:
Change the variable name `url2` to a more descriptive name.

Reply:
Ok
I will